mypy/fastparse.py

```diff
@@ -861,7 +865,9 @@ def is_star2arg(k: ast3.keyword) -> bool:
 
     # Num(object n) -- a number as a PyObject.
     @with_line
-    def visit_Num(self, n: ast3.Num) -> Union[IntExpr, FloatExpr, ComplexExpr]:
+    def visit_Num(self, nx: ast3.Num) -> Union[IntExpr, FloatExpr, ComplexExpr]:
+        # XXX: the typeshed stubs are lies
+        n = nx  # type: Any
```

Comment:
I think this was recently fixed. Did you sync typeshed? Or if stubs are still bad, then maybe we can just fix them.

Reply:
Oh oops, I had meant to remove this. I did indeed just fix the typeshed stub for it.